tests/parametric/test_config_consistency.py

```diff
@@ -461,12 +467,12 @@ def test_default_config(self, test_library, path, library_env, name, apm_configu
         [
             {
                 "apm_configuration_default": {
-                    "DD_RUNTIME_METRICS_ENABLED": True,
+                    "DD_LOGS_INJECTION": True,
                     "DD_FOOBAR_ENABLED": "baz",
                 },
                 "expected": {
                     **SDK_DEFAULT_STABLE_CONFIG,
-                    "dd_runtime_metrics_enabled": "true",
+                    "dd_logs_injection": "true",
```

Comment:
Can you explain this change? Is it because PHP does not support runtime metrics, so we're testing another config that we expect to be supported across all libs, i.e, logs_injection?
I ask because this will put us in another bind: Go does not support `DD_LOGS_INJECTION` [yet], and supporting it would be out of scope for stable config.

Reply:
You're correct; it seems we're in a catch 22 lol. Maybe we can find a commonly supported field? cc @iamluc